scripts/publish

```diff
@@ -15,6 +15,12 @@ if ! command -v "${PREFIX}twine" &>/dev/null ; then
     exit 1
 fi
 
+if ! ${PREFIX}pip show wheel &>/dev/null ; then
```

Comment:
The check is slightly different from the rest since in MacOS for example `/usr/local/bin/wheel` exists, but the build process was failing because the virtualenv did not have the package installed.

Reply:
Thanks for the clarification!